docs/DeveloperGuide.md

```diff
@@ -313,287 +313,289 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **Use case ID: UC01**
 
-**Use case: Create fileInfo**
+**Use case: Tag a file**
 
 **MSS**
 
-1. User requests to tag a certain file using the absolute file path.
-2. HelloFile recognises the file path to be valid.
-3. HelloFile creates the tag and display the fileInfo.
+1. User requests to tag a file with a tag name, the file path and a label.
+2. HelloFile creates the tag and informs the user.
 
     Use case ends.
 
 **Extensions**
 
-* 2a. HelloFile detects the file path is invalid.
+* 2a. HelloFile detects the tag name entered to be invalid.
 
-    * 2a1. HelloFile prompts the user that the file path is wrong.
+    * 2a1. HelloFile prompts the user that the tag name entered is invalid.
 
         Use case resumes from step 1.
 
-* 2b. HelloFile detects no tag input.
+* 2b. HelloFile detects the file path entered to be invalid.
 
-    * 2b1. HelloFile prompts the user for a tag.
+    * 2b1. HelloFile prompts the user that the file path entered is invalid.
 
         Use case resumes from step 1.
+        
+* 2c. HelloFile detects the label entered to be invalid.
```

Comment:
to be invalid -> is invalid

Reply:
emmm, is to be invalid wrong here? I'm not sure.

Reply:
Yup it's a grammatical mistake

Reply:
Ok, changed